pkg/clusteragent/admission/mutate/autoinstrumentation/auto_instrumentation_test.go

```diff
@@ -199,7 +199,7 @@ func TestInjectAutoInstruConfig(t *testing.T) {
 			pod:  common.FakePodWithEnvFieldRefValue("dotnet-pod", "CORECLR_PROFILER", "path"),
 			libsToInject: []libInfo{
 				{
-					lang:  dotnet,
+					lang:  "dotnet",
```

Comment:
nitpick: these should be the `language` types (unnecessary change)

Reply:
I've undone these now 👍 
I remember that I noticed them when merging some conflicting changes from the main branch, and these two were the only tests in the array that wasn't double quoted, see [this snippet](https://github.com/DataDog/datadog-agent/blob/bded3c7876f8ba18907c8a62f51a041b78d619fc/pkg/clusteragent/admission/mutate/autoinstrumentation/auto_instrumentation_test.go#L185-L233), so I thought I'd "fix" it for consistency 😅
I can leave it as-is or properly fix the rest of the tests in the array if you'd like?